e2e-tests/mdx/cypress/support/commands.js

```diff
@@ -0,0 +1,3 @@
+Cypress.Commands.add(`assertRoute`, (route) => {
+  cy.url().should(`equal`, `${window.location.origin}${route}`)
```

Comment:
We're used to do this
```suggestion
  cy.location(`pathname`).should(`eq`, route)
```

Reply:
Also why abstract it away? It's a native supported command?

Reply:
We're not actually using it, so I think it's ok to remove. I brought it over from the setup the themes repo is using on instinct. I'll nuke it.

Reply:
YAGNI :) THanks